chtls.c

```diff
@@ -1436,7 +1435,7 @@ forward_on_tls(struct chroute *rt, u_short schad, u_short dchad, struct chaos_he
 	 (rt->rt_braddr == 0 && (tlsdest[i].tls_addr == rt->rt_dest))
 	 ||
 	 // multiplexed
-	 is_in_mux_list(dchad, &tlsdest[i].tls_muxed)
+	 is_in_mux_list(dchad, tlsdest[i].tls_muxed)
```

Comment:
The warning here was that the original code refers to an address of an array.

Reply:
Thanks, this was the only warning I saw without -Wall.